Applications/Basic/CaptureHDRLoop/CaptureHDRLoop.cpp

```diff
@@ -0,0 +1,100 @@
+/*
+This example shows how to acquire HDR images from the Zivid camera in a loop
+(while actively changing some HDR settings). Two methods are demonstrated:
+the direct HDR capture, which is faster, and the indirect method, which is
+slower but preserves single captures.
+*/
+
+#include <Zivid/CloudVisualizer.h>
+#include <Zivid/Zivid.h>
+
+#include <iostream>
+
+int main()
+{
+    try
+    {
+        Zivid::Application zivid;
+
+        std::cout << "Setting up visualization" << std::endl;
+        Zivid::CloudVisualizer vis;
+        zivid.setDefaultComputeDevice(vis.computeDevice());
+
+        std::cout << "Connecting to the camera" << std::endl;
+        auto camera = zivid.connectCamera();
+
+        std::cout << "Acquiring HDR frame directly (faster)" << std::endl;
+        std::vector<Zivid::Settings> settingsVector;
+
+        size_t iris[3] = { 10U, 20U, 30U };
+        int exposureTime[3] = { 10000, 20000, 30000 };
+
+        for(size_t i = 0; i < 3; ++i)
+        {
+            settingsVector.push_back(
+                Zivid::Settings{}
+                    .set(Zivid::Settings::Iris{ iris[i] })
+                    .set(Zivid::Settings::ExposureTime{ std::chrono::microseconds{ exposureTime[i] } })
+                    .set(Zivid::Settings::Brightness{ 1 })
+                    .set(Zivid::Settings::Gain{ 1 })
+                    .set(Zivid::Settings::Bidirectional{ false })
+                    .set(Zivid::Settings::Filters::Contrast::Enabled::yes)
```

Comment:
If the goal of this example is to demonstrate HDR captures, I don't think we need to set all these settings. It just complicates the sample. I know it was like that before too, but maybe we could remove most of them for both the old and the new code.

Reply:
That is the part of what we want to show in this sample, i.e. how to set all possible settings. We want to allow the users to just copy-paste what they need from the settings.

We have removed the sample CaptureHDRCompleteSettings and I am trying to add that code in this sample.

Perhaps the description should be modified instead to specify why we are configuring complete settings.

Reply:
Aren't "How do I make an HDR capture" and "What's the code for setting all possible settings" two different questions which would better be explained by two different samples?

> We have removed the sample CaptureHDRCompleteSettings

When was that removed?

Reply:
'How do I make an HDR capture' is answered in [CaptureHDR](https://github.com/zivid/cpp-extra-samples/blob/master/Camera/Basic/CaptureHDR/CaptureHDR.cpp) (this one should only have Zivid::HDR::capture in my opinion because that is the fast method but that is a different story)

This example aims to give a bit more than just how to capture an HDR, it aims to demonstrate a) two different ways of taking an HDR, b) changing settings before and in a loop, c) configuring all settings.

We started having too many samples so it was decided that CaptureHDRCompleteSettings was to be coved by CaptureHDRLoop, and it was removed in this [commit](https://github.com/zivid/cpp-extra-samples/commit/84990f969440d0b1a1b516391ed78a19c2a56000).

Maybe we should sit and decide how we deal with a) b) c). @torbsorb @knatten?

Reply:
I would probably have done it differently, but if you want to do all of this in this sample, that's up to you. You have a better overview of what needs doing and what's done elsewhere than I do.

Regarding removal of `CaptureHDRCompleteSettings`: I think it's important to note in commit messages if you remove stuff. "Reorganize samples" sounds like samples were just moved around. If you deleted stuff or consolidated some samples in that commit, it would have been good if it was mentioned in the message. 

Sitting down together and looking at this sounds like a good idea.